Assets/Scripts/Localization/LocalizationTable.cs

```diff
@@ -114,6 +114,7 @@ public static void SetLocalization(int lang)
         {
             string[] languages = GetLanguages();
             currentLanguage = languages[lang];
+            Debug.LogWarning(languages[lang]);
```

Comment:
Doesn't need to be a warning (just a general) and can probably be the new UnityDebugger rather than just Debug.Log